vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -93,11 +89,9 @@ $global:map = @{
     # specified by AKS PR for most of the cases. BUT as long as there's a new unpacked image version, we should keep the
     # versions synced.
     "c:\akse-cache\containerd\"   = @(
-        "https://acs-mirror.azureedge.net/containerd/windows/v0.0.41/binaries/containerd-v0.0.41-windows-amd64.tar.gz",
         $defaultContainerdPackageUrl
     );
     "c:\akse-cache\csi-proxy\"    = @(
-        "https://acs-mirror.azureedge.net/csi-proxy/v0.2.2/binaries/csi-proxy-v0.2.2.tar.gz",
```

Comment:
pls keep `csi-proxy-v0.2.2.tar.gz` since aks is still using `csi-proxy-v0.2.2.tar.gz` by default now, there is a work item to track